examples/support-agent/src/actions/createTicket.ts

```diff
@@ -1,23 +1,67 @@
 import { createAction } from "spinai";
 
+// Potentially problematic - hardcoded values and no error handling
+const PRIORITY_KEYWORDS = {
+  urgent: ["urgent", "emergency", "critical", "broken"],
+  high: ["important", "error", "failed"],
+  medium: ["issue", "problem", "bug"],
+  low: ["question", "help", "how to"]
+};
+
```

Comment:
The priority calculation logic is hardcoded. Consider making it configurable or more dynamic.